armory/baseline_models/pytorch/carla_goturn.py

```diff
@@ -3,6 +3,16 @@
 import numpy as np
 from art.estimators.object_tracking import PyTorchGoturn
 
+# TODO Fix this Implementation either pull code into armory direct
+#  or work with goturn dev to get proper release
+from armory.utils import external_repo
+from armory import paths
+
+runtime_paths = paths.runtime_paths()
+external_repo_dir = runtime_paths.external_repo_dir
+external_repo.download_and_extract_repos(
```

Comment:
This may conflict or overwrite downloads from the scenario config loading. See my recent PR for an alternative.